http-client/src/main/java/no/nav/familie/http/ecb/ECBRestClient.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.familie.http.ecb
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.http.config.ECBRestTemplate
+import no.nav.familie.http.ecb.domene.ECBExchangeRatesData
+import no.nav.familie.http.ecb.domene.ExchangeRate
+import no.nav.familie.http.ecb.domene.toExchangeRates
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.context.annotation.Import
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestOperations
+import java.net.URI
+import java.time.LocalDate
+
+@Component
+@Import(ECBRestTemplate::class)
+class ECBRestClient(@Qualifier("ecbRestTemplate") private val restOperations: RestOperations) : AbstractRestClient(restOperations, "ecb") {
+
+    private final val ECBApiUrl = "https://sdw-wsrest.ecb.europa.eu/service/data/EXR/"
```

Comment:
Er det noko poeng i å trekkje ut denne som parameter? Ikkje sikkert det er så farleg, sia det er eit opent API og vi berre hentar inn.

Reply:
Du tenker noe sånt? 

```
class ECBRestClient(@Qualifier("ecbRestTemplate") private val restOperations: RestOperations, @Value("\${ECB_API_UL}")private val ecbApiUrl: String)
```

Har vurdert det 🤔 Det blir en ekstra ting som brukere av lib'en må vite om og forholde seg til, men det er kanskje litt bedre praksiss?

Reply:
> 

Ja, stemmer, ser for meg noko sånt. Kan eventuelt bruke dagens adresse som standard-verdi, sånn at ein som brukar ikkje treng å gjera noko med mindre ein vil overstyre.